haskell/private/actions/compile.bzl

```diff
@@ -213,7 +213,17 @@ def _compilation_defaults(hs, cc, java, dep_info, plugin_dep_info, srcs, import_
             set.mutable_insert(import_dirs, idir)
 
     ghc_args += ["-i{0}".format(d) for d in set.to_list(import_dirs)]
-    ghc_args += ["-optP" + f for f in cc.cpp_flags]
+
+    # Write the -optP flags to a response because they can be very long on Windows
```

Comment:
What do you mean by a "response"?

Reply:
[response file](https://github.com/bazelbuild/bazel/issues/3750).

Reply:
If you like this kind of dialog: https://github.com/bazelbuild/bazel/issues/3750#issuecomment-330201248

tl;dr; For me a response file is a file which contains response to questions, for an automatic installation processes for example. Perhaps the term in not precise enough and may surprise future developers.

Reply:
I called it a parameter file in the comment, to match the Bazel terminology